research-2023/effort-responsibility-sharing/2023-03-08.003.md

```diff
@@ -0,0 +1,265 @@
+| Baseline Entity/Framework                      | While the "Baseline" most relates to FedRAMP, this could relate to other regulatory frameworks                                                       | FedRAMP                                                  |
+
+
+### Single Control Examples
+
+An example of an export using [Use Case 1: PE-11](2022-07-05.001.md#outcome), based on the approach [outlined in the previous sprial](2022-07-05.002.md#general-approach).
+
+```yaml
+control-implementation:
+ implemented-requirements:
+   - uuid: NISTDEMO-IR01-0000-0000-000000000000
+     control-id: pe-11
+     by-components:
+      - component-uuid: NISTDEMO-BC01-0000-0000-000000000000
+        uuid: NISTDEMO-CD01-0000-0000-000000000000
+        description: >-
+          The infrastructure is supported by multiple uninterruptible power supplies, and a redundant 
+          backup generator system capable of continuous operation for an extended period of time.
+        # Export Assembly
+        export:
+          description: Infrastructure as a Service - This is actually the redacted content to share for the control.
+          responsibilities:
+            - uuid: NISTDEMO-RES1-0000-0000-000000000000
+              provided-uuid: NISTDEMO-PRO1-0000-0000-000000000000
+              description: >-
+                Customer fully inherits this control from the Cloud Service Provider (CSP).
+              # Additional fields
+              import-ref: NISTDEMO-EXT01-0000-0000-000000000000
+              action: fully-inherit
+          provided:
+            - uuid: NISTDEMO-PRO1-0000-0000-000000000000
+              # Implementation Status Assembly
+              implementation-status:
+                - state: implemented
+                  remark: "Optional communication of intent"
+              description: >-
+                Customers do not have physical access to any system resources in Provider datacenters.
+```
+---
+
+An example based on [community feedback](https://github.com/usnistgov/OSCAL-DEFINE/discussions/11#discussioncomment-5336282) to consider an option that uses existing assemblies of the SSP model.  This assumes *common fields* with more structured information in the SSP would not typically contain sensitive information, for example, the implementation status of a control. Unstructured fields containing potentially sensitive information use the existing description fields in the export assembly.
+
+The concern is that any transformation of the SSP into a CRM-style model would have to follow a set of rules to prevent inadvertently publishing unintended information.  Placing all exportable content under the export, is a clear intent to publish customer-facing content.
+
+```yaml
+control-implementation:
+ implemented-requirements:
+   - uuid: NISTDEMO-IR01-0000-0000-000000000000
+     control-id: pe-11
+     by-components:
+      - component-uuid: NISTDEMO-BC01-0000-0000-000000000000
```

Comment:
Non-blocking comment: is this the demo UUID you were talking about? Because I love this.